modules/nf-core/blast/blastdbcmd/main.nf

```diff
@@ -0,0 +1,103 @@
+process BLAST_BLASTDBCMD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/blast:2.15.0--pl5321h6f7f691_1':
+        'biocontainers/blast:2.15.0--pl5321h6f7f691_1' }"
+
+    input:
+    tuple val(meta) , val(entry)
+    tuple val(meta2), path(db)
```

Comment:
Hi! Thanks for reviewing it. I was using two modules for either a string (`-entry`) or a file (`-entry_batch`) as input. If you know any other existing module that can deal with a similar scenario of mutually exclusive inputs, I would use it as a model. If not, I would split it into two.

Reply:
I swear I've added something along those lines at some point but I can't remember it right now (also on parental leave until Friday so I can't look into then), but otherwise probably the simplest would to split it in two to be honest. 

It would also be easier due to the stricter meta.yaml checks now on channel type descriptions 😅

Reply:
Just for sake of information, I noticed that `taxonkit/name2taxid` looks like a similar case to this regarding mutually exclusive inputs. Do we carry on with two modules, then @jfy133 ?

Reply:
Sorry haven't had a chance to look at this (still on half-time parental leave). I leave that up to you. Mututally exclusive inputs I Think would make a more concise and clearer module in this case, but might be more work for you now?

Reply:
No worries. Since it is not still merged, no problem with a bit more of work. From a user experience perspective, I have the feeling that it might be better keep everything in an only module. I will follow then `taxonkit/name2taxid` approach, thx!